### ISSUE TITLE
Module fails if you're using apollo-server-core and not apollo-server

### ISSUE THREAD

**Issue by robross0606** [profusion/apollo-validation-directives#20]
This module requires apollo-server but apparently doesn't declare as a dependency that it requires it.  I have a fully functional Apollo app that directly has `apollo-server-koa` and `apollo-server-core` as dependencies, but not `apollo-server`.  When I try to load this module, it fails:
```javascript
const {
  ValidateDirectiveVisitor,
  stringLength,
  range,
  pattern,
  listLength
} = require('@profusion/apollo-validation-directives')
```
```
2020-08-24T19:35:05.612Z [error]: uncaughtException: Cannot find module 'apollo-server'
Require stack:
- C:\Code\ncp\study-design-service\node_modules\@profusion\apollo-validation-directives\build\lib\EasyDirectiveVisitor.js
- C:\Code\ncp\study-design-service\node_modules\@profusion\apollo-validation-directives\build\lib\index.js
- C:\Code\ncp\study-design-service\src\graphql\schema\index.js
- C:\Code\ncp\study-design-service\src\graphql\index.js
- C:\Code\ncp\study-design-service\src\server.js
- C:\Code\ncp\study-design-service\index.js
Error: Cannot find module 'apollo-server'
Require stack:
- C:\Code\ncp\study-design-service\node_modules\@profusion\apollo-validation-directives\build\lib\EasyDirectiveVisitor.js
- C:\Code\ncp\study-design-service\node_modules\@profusion\apollo-validation-directives\build\lib\index.js
- C:\Code\ncp\study-design-service\src\graphql\schema\index.js
- C:\Code\ncp\study-design-service\src\graphql\index.js
- C:\Code\ncp\study-design-service\src\server.js
- C:\Code\ncp\study-design-service\index.js
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:965:15)
    at Function.Module._load (internal/modules/cjs/loader.js:841:27)
    at Module.require (internal/modules/cjs/loader.js:1025:19)
    at require (internal/modules/cjs/helpers.js:72:18)
    at Object.<anonymous> (C:\Code\ncp\study-design-service\node_modules\@profusion\apollo-validation-directives\build\lib\EasyDirectiveVisitor.js:15:25)
    at Module._compile (internal/modules/cjs/loader.js:1137:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1157:10)
    at Module.load (internal/modules/cjs/loader.js:985:32)
    at Function.Module._load (internal/modules/cjs/loader.js:878:14)
    at Module.require (internal/modules/cjs/loader.js:1025:19)
    at require (internal/modules/cjs/helpers.js:72:18)
    at Object.<anonymous> (C:\Code\ncp\study-design-service\node_modules\@profusion\apollo-validation-directives\build\lib\index.js:4:30)
    at Module._compile (internal/modules/cjs/loader.js:1137:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1157:10)
    at Module.load (internal/modules/cjs/loader.js:985:32)
    at Function.Module._load (internal/modules/cjs/loader.js:878:14)  metadata: {
  "error": {
    "code": "MODULE_NOT_FOUND",
    "requireStack": [
      "C:\\Code\\ncp\\study-design-service\\node_modules\\@profusion\\apollo-validation-directives\\build\\lib\\EasyDirectiveVisitor.js",
      "C:\\Code\\ncp\\study-design-service\\node_modules\\@profusion\\apollo-validation-directives\\build\\lib\\index.js",
      "C:\\Code\\ncp\\study-design-service\\src\\graphql\\schema\\index.js",
      "C:\\Code\\ncp\\study-design-service\\src\\graphql\\index.js",
      "C:\\Code\\ncp\\study-design-service\\src\\server.js",
      "C:\\Code\\ncp\\study-design-service\\index.js"
    ]
  }
```

**Comment by robross0606** [comment]
Also, the instructions in your readme appear to be incorrect.  This does not find the module at all:
```import { auth } from 'apollo-validation-directives';```
But this does:
```import { auth } from '@profusion/apollo-validation-directives';```

**Comment by cabelitos** [comment]
@robross0606 Hey, yeah.  maybe we should change our peer-dep to apollo-server-core and not apollo-server. I'll create a MR changing that. Thanks for the heads up.

**Comment by cabelitos** [comment]
fixed on 2.0.0 (available on NPM right now)
